x/bank/keeper/send.go

```diff
@@ -218,13 +222,9 @@ func (k BaseSendKeeper) SendCoins(ctx context.Context, fromAddr, toAddr sdk.AccA
 }
 
 // subUnlockedCoins removes the unlocked amt coins of the given account. An error is
-// returned if the resulting balance is negative or the initial amount is invalid.
+// returned if the resulting balance is negative.
```

Comment:
nit: here and `SendCoins`: some godoc comment about the amount assumptions would be helpful